hack/csv-generate.sh

```diff
@@ -4,18 +4,27 @@ set -e
 
 export GOROOT=$(go env GOROOT)
 
+PREV="4.4.0"
+LATEST="4.5.0"
+
 IS_DEV=$( [[ $1 = "-dev" ]] && echo true || echo false )
-CSV_VERSION=$( [[ $IS_DEV = true ]] && echo "0.0.1" || echo "$CSV_VERSION" )
+
+if [[ $IS_DEV = true ]] || [[ -z "$CSV_VERSION" ]]; then
+  CSV_VERSION=$LATEST
+  REPLACES_CSV_VERSION=$PREV
+fi
+
 
 PACKAGE_NAME="performance-addon-operator"
 
 PACKAGE_DIR="deploy/olm-catalog/${PACKAGE_NAME}"
 CSV_DIR="${PACKAGE_DIR}/${CSV_VERSION}"
-CSV_FILE="$CSV_DIR/${PACKAGE_NAME}.v${CSV_VERSION}.clusterserviceversion.yaml"
 
 OUT_DIR="build/_output/olm-catalog"
 OUT_CSV_DIR="${OUT_DIR}/${PACKAGE_NAME}/${CSV_VERSION}"
-OUT_CSV_FILE="${OUT_CSV_DIR}/${PACKAGE_NAME}.v${CSV_VERSION}.clusterserviceversion.yaml"
+
+TMP_CSV_VERSION="999.999.999"
```

Comment:
I do not understand why we need this temporary CSV :thinking:

Reply:
removed tmp folder